tests/AutoReview/ProjectCodeTest.php

```diff
@@ -82,7 +83,7 @@ public function testThatSrcClassHaveTestClass($className)
         }
 
         $this->assertTrue(class_exists($testClassName), sprintf('Expected test class "%s" for "%s" not found.', $testClassName, $className));
-        $this->assertTrue(is_subclass_of($testClassName, '\PHPUnit_Framework_TestCase'), sprintf('Expected test class "%s" to be a subclass of "\PHPUnit_Framework_TestCase".', $testClassName));
+        $this->assertTrue(is_subclass_of($testClassName, '\PHPUnit\Framework\TestCase'), sprintf('Expected test class "%s" to be a subclass of "\PHPUnit\Framework\TestCase".', $testClassName));
```

Comment:
no.

https://github.com/sebastianbergmann/phpunit/blob/4.8/src/ForwardCompatibility/TestCase.php

is something is subclass of `PHPUnit_Framework_TestCase`, then it may not be subclass of `PHPUnit\Framework\TestCase`. we expect later, so we check for later. if one would implement test using first class, this assertion would fail.